app/javascript/controllers/image_controller.js

```diff
@@ -0,0 +1,55 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="image"
+export default class extends Controller {
+    static targets = ['input', 'preview', 'previewArea']
+
+    preview() {
+        const input = this.inputTarget;
```

Comment:
```suggestion
    preview(e) {
        const files = e.currentTarget.files;
```

このようにevent経由で要素を参照できるのでinputのtagetsを登録する必要はないかも？？

Reply:
ですね！ありがとうございます。